### ISSUE TITLE
🎢 Update form  field input copy on "add an experience" page

### ISSUE THREAD

**Issue by RM-1978** [GCTC-NTGC/gc-digital-talent#7007]
The Recruitment team is not receiving enough experience description to conduct their assessments with this in put guidance below: 

![Image](https://github.com/GCTC-NTGC/gc-digital-talent/assets/101275838/ffaa62b6-0c15-4259-a3e2-46da7d968bb5)

This request may be resolved once we implement the complete skills libraries so that candidates can demonstrate their skills against the skills requirements for our posters.   

Potential Solutions: 

As a short-term solution, we could update the input text with the copy (and the accordion) we use in the skill requirement modal to suggest how users can best prepare their content earlier in the process: 
![Image](https://user-images.githubusercontent.com/101275838/246892316-1cc6730b-0bd8-4646-8e09-96e8013ee95e.png)

**Comment by RM-1978** [comment]
### Proposed Solution 

@gobyrne and @substrae, 

To respond to David Dufour's pain point, what do you think about not hiding the _How best to describe a skill experience_ instruction in an accordion?  He also proposed directional language for an update to this copy, but I am inclined to simply propose that we update to the following text: 

`Help us find your best match by answering the questions below in your description.` 

@esizer or @mnigh is this too directional, or have i successfully ridden the line?

**Comment by esizer** [comment]
> `Help us find your best match by answering the questions below in your description.`

This sounds good to me, the only tweak I would make would be the update it to "...answering the _following_ questions..."

**Comment by gobyrne** [comment]
Are we trying to improve the input applicants are putting the "Task and Responsibilities" section of the experience form, or the "skill justification" section of the linking form?

**Comment by substrae** [comment]
@RM-1978 @gobyrne 

I'm confused by this issue. 

> The Recruitment team is not receiving enough experience description to conduct their assessments

If this is actually the case, then this is a problem with the skill justification step during the application process. There could be many things going wrong here though:
- Is RATA trying to assess skills that aren't asked for on the poster?
- Is RATA unable to assess skills that ARE asked for on the poster because the justification and linked experience aren't enough?
- What category of skills (behavioural, technical) are we talking about here?

> This request may be resolved once we implement the complete skills libraries so that candidates can demonstrate their skills against the skills requirements for our posters.

This is already the case - if RATA has included the skill on the poster, then the candidate can demonstrate against that skill during the application by linking it to experience.

I don't think this has anything to do with the "Tasks and responsibilities" field, which, by the way, is being updated as a part of the profile updates anyway.

**Comment by gobyrne** [comment]
Let's reach back out to the recruitment team on this one. I need more info about the problem.

**Comment by gobyrne** [comment]
Ok, recruitment team says this is behaving reasonably for now. They are also optimistic that skill levels will improve things as well. We're always interested in improving this data, but nothing concrete here. Closing issue for now.

**Comment by RM-1978** [comment]
I think the design team started ideating solutions as soon as he heard David's feedback on May 5, but this issue was surfaced a few releases ago. 

Since this issue was received, we have changed the instructional copy under another issue since then to address these observations:

![14ce6884-c48f-4178-b969-d88bed26b491](https://github.com/GCTC-NTGC/gc-digital-talent/assets/101275838/7230c2e6-256a-4b1e-9b83-f75910744d11)


Here is what we now say on adding an experience to the career timeline and recruitment section of the Dashboard: 
 
![f7d7b958-88aa-410e-8fc4-8c96cde2fc43](https://github.com/GCTC-NTGC/gc-digital-talent/assets/101275838/deace6d0-f5e6-49c0-8bb5-0f0a762a1394)
